site/content/posts/2023-08-08-v0.25.0-impersonation-proxy-with-external-certs.md

```diff
@@ -0,0 +1,318 @@
+---
+title: "Pinniped v0.25.0: With External Certificate Management for the Impersonation Proxy and more"
+slug: v0-25-0-external-cert-mgmt-impersonation-proxy
+date: 2023-08-08
+author: Joshua T. Casey
+image: https://images.unsplash.com/photo-1618075254460-429d47b887c7?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2148&q=80
+excerpt: "With v0.25.0 you get external certificate management for the impersonation proxy, easier scheduling of the kube-cert-agent, and more"
+tags: ['Joshua T. Casey','Ryan Richard', 'Benjamin Petersen', 'release']
+---
+
+![Friendly seal](https://images.unsplash.com/photo-1618075254460-429d47b887c7?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2148&q=80)
+*Photo by [karlheinz_eckhardt Eckhardt](https://unsplash.com/@karlheinz_eckhardt) on [Unsplash](https://unsplash.com/s/photos/seal)*
+
```

Comment:
Lets bubble up mTLS to the top here in the summary section since it is part of the reason this feature is being added (not excessively, but a short explanation that Concierge provides certificates to users to begin with, and mTLS is the mechanism for used for users to login with these certs).

Reply:
> To read more about the impersonation proxy, see the [docs](https://pinniped.dev/docs/reference/supported-clusters/#background).

I didn't want to add docs to the blog, so I punted with a link.